hr.ibicom.treemly.yml

```diff
@@ -0,0 +1,25 @@
+app-id: hr.ibicom.treemly
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: treemly
+finish-args:
+  - --socket=fallback-x11
```

Comment:
how is it gonna display anything on a wayland session?

Also missing `--share=ipc`.